Documentation/gettingstarted/k8s-install-openshift-okd.rst

```diff
@@ -0,0 +1,252 @@
+   bootstrapping to complete...`` appears in the logs. It is safe to apply
+   these changes once, OpenShift will not override these.
+
+.. tabs::
+
+   .. group-tab:: AWS: enable Cilium ports 
+
+      .. code:: bash
+
+         infraID="$(jq -r < "${CLUSTER_NAME}/metadata.json" '.infraID')"
+         aws_region="$(jq -r < "${CLUSTER_NAME}/metadata.json" '.aws.region')"
+         cluster_tag="$(jq -r < "${CLUSTER_NAME}/metadata.json" '.aws.identifier[0] | to_entries | "Name=tag:\(.[0].key),Values=\(.[0].value)"')"
+
+         worker_sg="$(aws ec2 describe-security-groups --region "${aws_region}" --filters "${cluster_tag}" "Name=tag:Name,Values=${infraID}-worker-sg" | jq -r .SecurityGroups[0].GroupId)"
+         master_sg="$(aws ec2 describe-security-groups --region "${aws_region}" --filters "${cluster_tag}" "Name=tag:Name,Values=${infraID}-master-sg" | jq -r .SecurityGroups[0].GroupId)"
+
+         aws ec2 authorize-security-group-ingress --region "${aws_region}" \
+            --ip-permissions \
+               "IpProtocol=udp,FromPort=8472,ToPort=8472,UserIdGroupPairs=[{GroupId=${worker_sg}},{GroupId=${master_sg}}]" \
+               "IpProtocol=tcp,FromPort=4240,ToPort=4240,UserIdGroupPairs=[{GroupId=${worker_sg}},{GroupId=${master_sg}}]" \
+            --group-id "${worker_sg}"
+
+         aws ec2 authorize-security-group-ingress --region "${aws_region}" \
+            --ip-permissions \
+               "IpProtocol=udp,FromPort=8472,ToPort=8472,UserIdGroupPairs=[{GroupId=${worker_sg}},{GroupId=${master_sg}}]" \
+               "IpProtocol=tcp,FromPort=4240,ToPort=4240,UserIdGroupPairs=[{GroupId=${worker_sg}},{GroupId=${master_sg}}]" \
+            --group-id "${master_sg}"
+
+   .. group-tab:: GCP: enable Cilium ports
+
+      .. code:: bash       
+
+         infraID="$(jq -r < "${CLUSTER_NAME}/metadata.json" '.infraID')"
+         gcp_projectID="$(jq -r < "${CLUSTER_NAME}/metadata.json" '.gcp.projectID')"
+
+         gcloud compute firewall-rules create \
+            --project="${gcp_projectID}" \
+            --network="${infraID}-network" \
+            --allow=tcp:4240,udp:8472,icmp \
+            --source-tags="${infraID}-worker,${infraID}-master" \
+            --target-tags="${infraID}-worker,${infraID}-master" \
+              "${infraID}-cilium"
+
+Accessing the Cluster
+=====================
+
+To access the cluster you will need to use ``kubeconfig`` file from the ``${CLUSTER_NAME}/auth`` directory:
+
+.. code:: bash
+
+   export KUBECONFIG="${CLUSTER_NAME}/auth/kubeconfig"
+
+Prepare cluster for Cilium connectivity test
+============================================
+
+In order for Cilium connectivity test pods to run on OpenShift, a simple custom ``SecurityContextConstraints``
+object is required. It will to allow ``hostPort``/``hostNetwork`` that some of the connectivity test pods rely on,
+it sets only ``allowHostPorts`` and ``allowHostNetwork`` without any other privileges.
```

Comment:
Add a sentence like this: You can remove this ``SecurityContextConstraints`` again as you remove the connectivity test deployment from the cluster.

Reply:
Actually, I'm add a cleanup section and show exact command there.